aws/ecs_cluster.go

```diff
@@ -11,14 +11,63 @@ import (
 	"github.com/gruntwork-io/gruntwork-cli/errors"
 )
 
+// Used in this context to determine if the ECS Cluster is ready to be used & tagged
+// For more details on other valid status values: https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#Cluster
+const activeEcsClusterStatus string = "ACTIVE"
+const provisioningEcsClusterStatus = "PROVISIONING"
+
+// filter out all active ecs clusters
```

Comment:
NIT: I believe "filter out" means "get rid of", so this would be the opposite of what this function is doing.